### PR TITLE
[WIP] feat: inactive users index

### PR DESCRIPTION
**Pull request by pajgo**
Sub task for inactive users delete

### CODE REVIEW

File: src/utils/inactiveUsers/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const {
```
Comment (AVVS) [inline src/utils/inactiveUsers/index.js]: this file seems redundant and its better to just inline these 2 functions

File: src/actions/register.js [hunk 2]
```diff
@@ -208,7 +210,11 @@ async function performRegistration({ service, params }) {
   pipeline.hset(USERS_USERNAME_TO_ID, username, userId);
 
   if (activate === false && config.deleteInactiveAccounts >= 0) {
+    /* TODO Remove this line when last task in group merged */
     pipeline.expire(userDataKey, config.deleteInactiveAccounts);
+
+    /* Add user id to the inactive users index */
+    addToInactiveUsers(pipeline, userId, audience);
```
Comment (AVVS) [inline src/actions/register.js]: inline function call

**Comment by AVVS** [comment]
this lacks a method that will clean up inactive index of user ids that had expired and never completed activation - needs to be added before the PR can be merged

**Comment by pajgo** [comment]
> this lacks a method that will clean up inactive index of user ids that had expired and never completed activation - needs to be added before the PR can be merged

Clean up method planned in Parent PR which contains delete user procedure. It's a transitional PR. Maybe It should be removed and code migrated into parent PR?

File: src/utils/inactiveUsers/index.js (hunk 1 above)
Comment (pajgo) [reply src/utils/inactiveUsers/index.js]: Removed one-liners and just added their contents into corresponding places.

**Comment by pajgo** [comment]
Closing as this PR changes merged into #430

**Pull request closed without merging**